server/src/fakedb_writer.js

```diff
@@ -0,0 +1,62 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const stripBom = require('strip-bom-stream');
+
+const {createStore} = require('./database');
+
+// CSV file data
+const users = [];
+const events = [];
+const schedules = [];
+const tags = [];
+const userParticipatedEvents = [];
+
+const write_fake_db = async () => {
+  const db = createStore(true);
+  await db.sequelize.sync({force: true});
+  // The order of items MUST be stayed.
+  const read_info = [
+    {csv: 'User.csv', dat: users, db: db.User},
+    {csv: 'Event.csv', dat: events, db: db.Event},
+    {csv: 'Schedule.csv', dat: schedules, db: db.Schedule},
+    {csv: 'Tag.csv', dat: tags, db: db.Tag},
+    {
+      csv: 'UserParticipatedEvent.csv',
+      dat: userParticipatedEvents,
+      db: db.UserParticipatedEvent,
+    },
+  ];
+
+  for (let j = 0; j < read_info.length; j++) {
+    let fd = fs.createReadStream('../fakedb/' + read_info[j].csv).
```

Comment:
Any constant string in the code should be declared as a block at the top of the file, and follow the style guide to call it something like FAKEDB_PATH. See:

https://google.github.io/styleguide/javascriptguide.xml?showone=Constants#Constants